vdom/src/com/vdom/core/Player.java

```diff
@@ -791,13 +791,12 @@ public int getAllCardCount() {
         		if (actionCount >= 3) {
         			threePlusCopyActionCards++;
         		}
+			if (actionCount >= highestActionCardCount) {
+				secondHighestActionCardCount = highestActionCardCount;
+				highestActionCardCount = actionCount;
+			}
 			if (actionCount > secondHighestActionCardCount) {
```

Comment:
Why isn't this 'else if'? actionCount will be assigned to both highestActionCardCount and secondHighestActionCardCount and that doesn't seem like the intended behavior.